docs/pages/web/iconbuttonfloating.js

```diff
@@ -0,0 +1,248 @@
+              selected={selected}
+            />
+          </Dropdown.Section>
+        </Dropdown>
+      )}
+    </Flex>
+  );
+}
+
+      `}
+      />
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+            - To represent a primary or common action when it has to be visible all the time on the screen on top of everything.
+            - Triggering a Modal or a Popover to complete a related task.
+            - Only if it is the most suitable way to present a screen's high-emphasis action.
+        `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+            - There isn't a need for a fixed IconButtonFloating visible all the time on the screen.
+            - To replace [IconButton](/web/iconbutton) established patterns, such as nav elements.
+        `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use when an action has to be visible at all times in a sticky way where content can scroll underneath."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Layer notification badges on top of IconButtonFloating. This pattern is typically used on IconButtons part of a nav component, and IconButtonFloating shouldn't contain notifications found elsewhere on a screen, as it can lead to cognitive and usability issues. Users with color-blinded vision could also miss it since it doesn't offer a visually supportive affordance besides color."
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use IconButtonFloating for positive and supportive actions like Create, Help or Maximize."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use IconButtonFloating for negative and destructive actions like Delete or Remove. "
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use the designated size and style."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Scale or style IconButtonFloating. Consistent button sizes promote a cohesive user experience."
```

Comment:
This is more intended for designers... As an eng you can't modify tis component's style